server/app/services/geo/esri/EsriClient.java

```diff
@@ -144,11 +144,28 @@ CompletionStage<Optional<JsonNode>> fetchAddressSuggestions(ObjectNode addressJs
         Optional.ofNullable(addressJson.findPath(AddressField.STATE.getValue()).textValue());
     Optional<String> postal =
         Optional.ofNullable(addressJson.findPath(AddressField.ZIP.getValue()).textValue());
-    address.ifPresent(val -> request.addQueryParameter("address", val));
-    address2.ifPresent(val -> request.addQueryParameter("address2", val));
-    city.ifPresent(val -> request.addQueryParameter("city", val));
-    region.ifPresent(val -> request.addQueryParameter("region", val));
-    postal.ifPresent(val -> request.addQueryParameter("postal", val));
+    String singleLineAddress = "";
+    if (address.isPresent()) {
+      singleLineAddress += address.get();
+    }
+
+    if (address2.isPresent()) {
+      singleLineAddress += " " + address2.get();
+    }
+
+    if (city.isPresent()) {
+      singleLineAddress += " " + city.get();
+    }
+
+    if (region.isPresent()) {
+      singleLineAddress += " " + region.get();
+    }
+
+    if (postal.isPresent()) {
+      singleLineAddress += " " + postal.get();
+    }
```

Comment:
Does the address need to be formatted with commas to separate city, state, zip?

Reply:
It doesn't seem to matter, but I added a comma around city.